pulp_ansible/app/serializers.py

```diff
@@ -194,9 +194,11 @@ class CollectionVersionSerializer(SingleArtifactContentSerializer, ContentChecks
         child=serializers.CharField(max_length=64),
     )
 
-    contents = serializers.JSONField(help_text=_("A JSON field with data about the contents."))
+    contents = serializers.ListField(
+        child=serializers.DictField(), help_text=_("A JSON field with data about the contents.")
```

Comment:
works for:
`"content": [{"key1": "value1"}, {"key2": "value2"}]`
doesn't work for nested like:
`"content": [{"key1": {"key1-1": "value1-1"}}, {"key2": "value2"}]`